docs/getting_started/index.rst

```diff
@@ -0,0 +1,167 @@
+===============
+Getting Started
+===============
+The following directions will walk through the process of deploying Scribe and using the rsync functionality.
+In the future, this document will also contain steps for the rclone functionality.
+
+Deploying
+=========
+Once the repository has been cloned run the following within the scribe directory to deploy the required
+components.
+
+Volume SnapShot capabilities are required for Scribe. If you are using Kind to develop functionality for
+Scribe or another Kubernetes provider ensure that you are using a CSI capable storageclass.
+
+If you are wanting try scribe run the make command to deploy the objects into
+the cluster. Before starting ensure that your KUBECONFIG is set to the cluster you want to install Scribe.
+
+.. code-block:: bash
+
+   make deploy
+
+Verify Scribe is running.
+
+.. code-block:: bash
+
+   kubectl get pods -n scribe-system
+
+If you are developing and adding functionality to scribe run the following as it will output the logs of
+the controller to your terminal.
+
+.. code-block:: bash
+
+   make install
+   make run
+
+
+Using Rsync
+===========
+The following example will use the rsync replication method and take a Snapshot at the destination.
+A database will be used.
+
+First, create the destination and deploy the ReplicationDestination configuration.
+
+.. code-block:: bash
+
+   kubectl create ns dest
+   kubectl create -n dest -f examples/scribe_v1alpha1_replicationdestination.yaml
+
+A service object is created which will be used by the ReplicationSource to rsync the data. Record
```

Comment:
Instead of encouraging people to look at the Service, have them examine the ReplicationDestination `.status` to get the address (and the name of the keys).